t/static-files.t

```diff
@@ -371,13 +369,14 @@ ddc82658ac274d981a7e3159fc4ed452166f73a4  root/static/images/sponsors/idonethis.
 2b7374c1c82ad51415dacd32d033b119fcfa9cba  root/static/images/sponsors/open-cage.svg
 7f77555505dfcb1a2c3b47bec95062fe2dd7a5da  root/static/images/sponsors/panopta.png
 fb43b99a721b83aeebf528977f3df7083a69289e  root/static/images/sponsors/perl-careers.png
-1eacc5aedbbaeb388801652ecc48c76fce49d530  root/static/images/sponsors/perl-services.svg
 5cdd9d7bda9936c0ab678063a93df341fd37acb1  root/static/images/sponsors/perl_logo.png
+1eacc5aedbbaeb388801652ecc48c76fce49d530  root/static/images/sponsors/perl-services.svg
```

Comment:
A bunch of these position changes are due to sort being locale dependent. Could you maybe remove all of the changes except for the ones for the Diriv logo itself?

In the future, there will be a better way to update this file.

Reply:
OK :+1: